### ISSUE TITLE
AI does not disband Scrivener's Office emissary when there are 0 City-States

### ISSUE THREAD

**Issue by HeathcliffWarriors** [LoneGazebo/Community-Patch-DLL#5408]
_1. Mod version (i.e Date - 4/23):_ 6/2

_2. Mod list (if using Vox Populi only, leave blank):_ Archaeology Aesthetic Adjustments, Calypso's Colored Religious Icons, Quick Turns, IGE

_3. Error description:_ I set up a game where the AI's production and science costs were extremely low, but I set AIUnitCostPercent to 100%, so all units cost maintenance. There are 0 City-States in the game.

Despite this, upon building the Scrivener's Office, the Emissary is not disbanded after several turns, despite costing the AI unit maintenance (or if it's free, then a maintenance free unit slot).

_4. Steps to reproduce (optional):_ Set up a game with no City-States and wait for the AI to build the Scrivener's Office.

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

![Screenshot (13625)](https://user-images.githubusercontent.com/38794241/58780133-0404ce80-85a6-11e9-9149-04901331b668.png)
![Screenshot (13627)](https://user-images.githubusercontent.com/38794241/58780180-2860ab00-85a6-11e9-90d7-7fc921d42e47.png)
![Screenshot (13628)](https://user-images.githubusercontent.com/38794241/58780182-2ac30500-85a6-11e9-8d90-84619444f4a4.png)
![Screenshot (13632)](https://user-images.githubusercontent.com/38794241/58780187-2c8cc880-85a6-11e9-866f-027660efe5e1.png)
![Screenshot (13633)](https://user-images.githubusercontent.com/38794241/58780192-30b8e600-85a6-11e9-8244-96d92f3fb63c.png)

**Comment by HeathcliffWarriors** [comment]
I assume the same would likely apply to the free Great Diplomat from building the Roman Forum.

**Comment by LoneGazebo** [comment]
Yeah it won’t, typically, since civilian units have special circumstance deletion rules.

**Comment by HeathcliffWarriors** [comment]
UPDATE: I have added the HasMetValidMinorCiv function to the Diplo AI to aid in fixing this :)
